ui/app/components/app/menu-bar/index.scss

```diff
@@ -28,5 +29,12 @@
     cursor: pointer;
     display: flex;
     justify-content: center;
+    position: absolute;
+    right: 16px;
+    transform: rotate(90deg);
```

Comment:
Note that we already have three vertical dots as an icon in the project, via Font Awesome: https://fontawesome.com/icons/ellipsis-v?style=solid
I believe the intent was to use this icon. It does seem more appropriate than rotating this one.

Reply:
Yeah I agree, which was one of the reasons why I'd removed it from #8270—we can update it in a separate PR though